scale-tests/deploy_dsengine.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
```

Comment:
I don't see this in the `run.sh` script so for now this part is still a separate manual step separate from the run script, is that right?

Reply:
Correct. It's its own step in the runbook:
![image](https://user-images.githubusercontent.com/112426/66644465-478cda80-ec21-11e9-9342-2c00ed5c645b.png)